src/main/frontend/model/samples/migrate-resource-type.json

```
@@ -0,0 +1,46 @@
+{
+	"logLevel": "info",
+	"hops": [
+		{
+			"type": "declare",
+			"declarations": {
+				"sourceResourceType": "\"swisscom/components/content/accordion\"",
+				"destinationResourceType": "\"swisscom/components/content/accordion-new\""
+			}
+		},
+		{
+			"type": "nodeQuery",
+			"query": "SELECT * FROM [nt:unstructured] as component WHERE component.[sling:resourceType] = \"${sourceResourceType}\"",
```

You can make this a parameter of the prepared statement even:

```suggestion
			"query": "SELECT * FROM [nt:unstructured] as component WHERE component.[sling:resourceType] = $sourceResourceType",
```

---

Switched to parameters.